src/style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  margin: 0;
+}
+
+.block_container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 > * {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 > :first-child {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--1 > :nth-child(n+2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 > :nth-child(-n+2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 > :nth-child(n+3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 > :nth-child(-n+3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 > :nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 > :nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
+
+.stars--5 > * {
+  background-image: url("images/star-active.svg");
+
+}
```

Comment:
Try to improve this logic in the next way:
Let's add `background-image: url("images/star.svg");` for all stars (Move it to `.stars__star`) and apply active start image only to diagonal stars

Reply:
And after that, pay attention to 2-nd point from the [Checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)